src/Simplic.Data.Sql/SqlRepositoryBase.cs

```diff
@@ -60,24 +107,13 @@ public virtual TModel Get(TId id)
         /// <returns>Model if exists</returns>
         protected virtual TModel GetByColumn<T>(string columnName, T id)
         {
-            var key = $"{TableName}_{columnName}_{id}";
-
             TModel obj = default(TModel);
-            if (UseCache)
-            {
-                obj = cacheService.Get<TModel>(key);
-                if (obj != null)
-                    return obj;
-            }
 
             return sqlService.OpenConnection((connection) =>
             {
                 obj = connection.Query<TModel>($"SELECT * FROM {TableName} WHERE {columnName} = :id",
```

Comment:
Maybe we should use the query builder that dapper provides for better sql queries?

Reply:
@NexQueek is there an existing Sql Anywhere Provider?

Reply:
@simplicbe Its inside of dapper so if we use dapper, what we already do, it should work

Reply:
https://github.com/Drizin/DapperQueryBuilder

Reply:
Please create an issue for this. We should put it after a review in a separate PR.